### ISSUE TITLE
Test failing on HHVM

### ISSUE THREAD

**Issue by vkama** [Automattic/facebook-instant-articles-wp#701]
Adding HHVM to travis breaks the test:

`Notice: /tmp/wordpress-tests-lib/includes/../data/themedir1 is not readable in /tmp/wordpress/wp-includes/theme.php on line 471
Not running ajax tests. To execute these, use --group ajax.
Not running ms-files tests. To execute these, use --group ms-files.
Not running external-http tests. To execute these, use --group external-http.
PHPUnit 5.7.2 by Sebastian Bergmann and contributors.
F.                                                                  2 / 2 (100%)
Time: 27.17 seconds, Memory: 64.04MB
There was 1 failure:
1) Facebook\InstantArticles\Transformer\WPTransformerTest::testTransformerLikeWPContent
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
       <figure class="op-interactive">
-        <iframe class="no-margin">
-      <div>
-        <h1>Iframe loose without url</h1>
-      </div>
-    </iframe>
-      </figure>
-      <figure class="op-interactive">
-        <iframe class="no-margin">
-      <div>
-        <h1>some embed here</h1>
-      </div>
-    </iframe>
-      </figure>
-      <figure class="op-interactive">
-        <iframe class="no-margin">
-      <div>
-        <h1>some embed here</h1>
-      </div>
-    </iframe>
-      </figure>
-      <figure class="op-interactive">
-        <iframe class="no-margin">
-        <div>some content</div>
-        <script>alert('hi & hello to you @ testing!');</script></iframe>
-      </figure>
-      <figure class="op-interactive">
         <iframe src="http://example.com/4/" class="no-margin" width="300" height="200"></iframe>
       </figure>
       <figure class="op-interactive">
-        <iframe class="no-margin">
-        <div>some content</div>
-        <script>alert('hi & hello to you @ testing!');</script></iframe>
-      </figure>
-      <figure class="op-interactive">
         <iframe src="http://example.com/4/" class="no-margin" width="300" height="200"></iframe>
       </figure>
-      <figure class="op-interactive">
-        <iframe class="no-margin">
-        <div>some content</div>
-        <script>alert('hi & hello to you @ testing!');</script></iframe>
-      </figure>
-      <figure class="op-interactive">
-        <iframe class="no-margin" height="200">
-          <table width="200" height="200">
-            <thead><td>header 1</td>
-        <td>header 2</td>
-      </thead>
-            <tbody>
-              <tr><td>Line 1 column 1</td>
-          <td>Line 1 column 2</td>
-        </tr>
-              <tr><td>Line 2 column 1</td>
-          <td>Line 2 column 2</td>
-        </tr>
-            </tbody>
-          </table>
-        </iframe>
-      </figure>
@@ @@
         <iframe class="no-margin">
-          <blockquote class="instagram-media" data-instgrm-captioned="" data-instgrm-version="6" style=" background:#FFF; border:0; border-radius:3px; box-shadow:0 0 1px 0 rgba(0,0,0,0.5),0 1px 10px 0 rgba(0,0,0,0.15); margin: 1px; max-width:658px; padding:0; width:99.375%; width:-webkit-calc(100% - 2px); width:calc(100% - 2px);">
-      <div style="padding:8px;">
-        <div style=" background:#F8F8F8; line-height:0; margin-top:40px; padding:62.5% 0; text-align:center; width:100%;">
-          <div style=" background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAAsCAMAAAApWqozAAAAGFBMVEUiIiI9PT0eHh4gIB4hIBkcHBwcHBwcHBydr+JQAAAACHRSTlMABA4YHyQsM5jtaMwAAADfSURBVDjL7ZVBEgMhCAQBAf//42xcNbpAqakcM0ftUmFAAIBE81IqBJdS3lS6zs3bIpB9WED3YYXFPmHRfT8sgyrCP1x8uEUxLMzNWElFOYCV6mHWWwMzdPEKHlhLw7NWJqkHc4uIZphavDzA2JPzUDsBZziNae2S6owH8xPmX8G7zzgKEOPUoYHvGz1TBCxMkd3kwNVbU0gKHkx+iZILf77IofhrY1nYFnB/lQPb79drWOyJVa/DAvg9B/rLB4cC+Nqgdz/TvBbBnr6GBReqn/nRmDgaQEej7WhonozjF+Y2I/fZou/qAAAAAElFTkSuQmCC); display:block; height:44px; margin:0 auto -44px; position:relative; top:-22px; width:44px;"></div>
-        </div>
-        <p style=" margin:8px 0 0 0; padding:0 4px;">
-          <a href="https://www.instagram.com/p/BAXbKP1POQe/" style=" color:#000; font-family:Arial,sans-serif; font-size:14px; font-style:normal; font-weight:normal; line-height:17px; text-decoration:none; word-wrap:break-word;" target="_blank">ð¸ @natthaponwuttipetch</a>
-        </p>
-        <p style=" color:#c9c8cd; font-family:Arial,sans-serif; font-size:14px; line-height:17px; margin-bottom:0; margin-top:8px; overflow:hidden; padding:8px 0 7px; text-align:center; text-overflow:ellipsis; white-space:nowrap;">
-          A photo posted by Ann Hathairat Vidhyaphum (@annvidh) on <time style=" font-family:Arial,sans-serif; font-size:14px; line-height:17px;" datetime="2016-01-10T16:56:06+00:00">Jan 10, 2016 at 8:56am PST</time></p>
-      </div>
-    </blockquote>
@@ @@
         <iframe class="no-margin">
-      <h1>Sibling content</h1>
-      <div>sibling body</div><script src="//sibling.com/brother.js"></script></iframe>
+          <script src="//sibling.com/brother.js"></script>
+        </iframe>
       </figure>
       <p>Standard paragraph that <b>shouldn't</b> lie within the interactive block.</p>
       <figure class="op-interactive">
         <iframe class="no-margin">
           <p>Extra markup</p>
-          <div class="fb-post" data-href="https://www.facebook.com/some-page/posts/some-post"></div>
@@ @@
           <script type="text/javascript" src="//cdn.playbuzz.com/widget/feed.js"></script>
-          <div class="pb_feed" data-key="oembed" data-game="https://www.playbuzz.com/sportsillustrated/can-you-guess-the-olympic-sport-by-the-athlete-s-facial-expression" data-recommend="False" data-tags="ALL"></div>
         </iframe>
       </figure>
       <figure class="op-interactive">
         <iframe class="no-margin">
           <script type="text/javascript" src="//cdn.playbuzz.com/widget/feed.js"></script>
-          <div class="pb_feed" data-key="oembed" data-game="https://www.playbuzz.com/finnmorgan10/hugh-jackman-is-officially-saying-goodbye-to-wolverine" data-recommend="False" data-tags="ALL"></div>
@@ @@
           <script type="text/javascript" src="//cdn.playbuzz.com/widget/feed.js"></script>
-          <div class="pb_feed" data-key="oembed" data-game="http://www.playbuzz.com/valeriemack10/13-things-that-happen-to-people-who-are-learning-another-language" data-recommend="False" data-tags="ALL"></div>
         </iframe>
       </figure>
     </article>
   </body>
 </html>
 '
/home/travis/build/Automattic/facebook-instant-articles-wp/tests/Facebook/InstantArticles/Transformer/WPTransformerTest.php:74
FAILURES!
Tests: 2, Assertions: 2, Failures: 1.`

**Comment by vkama** [comment]
Check https://travis-ci.org/Automattic/facebook-instant-articles-wp/jobs/239577172#L454
for a more readable version

**Comment by timjacobi** [comment]
Test is passing, thanks for the commit @vkama
